tigerpath/majors_and_certificates/majors/COS-AB_2018.json

```diff
@@ -32,15 +37,12 @@
       "max_counted": 1,
       "min_needed": "ALL",
       "explanation": "The prerequisites for AB candidates are MAT 103, 104, and 202 or 204 or 217; COS 126, COS 217, and 226.\n\nStudents should plan to take both 217 and 226 before the junior year. One or both of these are required prerequisites for all later computer science courses.",
-      "double_counting_allowed": false,
-      "pdfs_allowed": false,
```

Comment:
Why is this being removed?
Was there a departmental policy change behind this allowing PDFing the prereqs? (That is, of course, besides that for the COVID19 pandemic, which says that all Spring 2020 courses can be PDFed, but that's a separate matter.)

We don't currently display this information, by the way, but we might in a future update.
Of course, we would never actually check whether or not a student in fact chooses to PDF a course. (How could we? We don't and can't see their grades; nor do we want to.) We would most likely just display that information to them in the popover. But then why not leave it in, so that it's there if/when we need it.

Reply:
I'd thought the default was the double-counting was false and PDF allowed is also false so I removed them here (and in other majors) because I thought it was redundant and slightly confusing when I was trying to keep track of the double-counting. If you'd prefer to have it back, let me know.

Reply:
Yeah, you're right, `double_counting_allowed` is false by default, so no need for it to be listed explicitly unless it's true, so I support removing that.

`pdfs_allowed`, on the other hand, doesn't really have a default yet as far as I know, because we haven't really settled yet on how to interpret and display it. Right now, if it's not listed, it mostly just means unspecified, I think.
It's not required to be listed, but if it's already there, no need to remove it, I think.
But, yeah, I'm not hung up on it, honestly. It's not an issue either way, since it has no real effect at the moment.